conftest.py

```diff
@@ -30,6 +30,11 @@ def pytest_addoption(parser):
                      action='store_true',
                      help='Log whether AWS API calls hit the cache. Requires -s')
 
+    parser.addoption('--aws-require-tag',
```

Comment:
NB: Would it be worth following the pattern of having the options be plural? 

Changing it to be:
```
--aws-require-tags Name,Type,App,Stack
```

Reply:
Yeah, they're inconsistent. I was thinking of making the other aws args (profiles and regions) singular, since the parser chokes on CSVs.

I'll make it plural and open an issue to discuss how we want to handle those args.

Reply:
Sounds good.

I've got a simple fix locally, which looks like this:

```
diff --git a/conftest.py b/conftest.py
index 3b1fe43..0247b21 100644
--- a/conftest.py
+++ b/conftest.py
@@ -30,6 +30,14 @@ def pytest_addoption(parser):
                      action='store_true',
                      help='Log whether AWS API calls hit the cache. Requires -s')
 
+def parse_opt(opt):
+    if not len(opt):
+        return None
+
+    if ',' in opt[0]:
+        return opt[0].split(',')
+
+    return opt
 
 def pytest_configure(config):
     global botocore_client
@@ -37,13 +45,7 @@ def pytest_configure(config):
     # monkeypatch cache.set to serialize datetime.datetime's
     patch_cache_set(config)
 
-    profiles, regions = config.getoption('--aws-profiles'), config.getoption('--aws-regions')
-
-    if not len(profiles):
-        profiles = None
-
-    if not len(regions):
-        regions = None
+    profiles, regions = parse_opt(config.getoption('--aws-profiles')), parse_opt(config.getoption('--aws-regions'))
```